core/app/components/pubs/PubEditor/PubEditorClient.tsx

```diff
@@ -171,12 +177,14 @@ export function PubEditorClient(props: Props) {
 			}
 
 			const result = await runCreatePub({
-				pubId,
+				body: {
+					id: pubId,
+					pubTypeId: pubTypeId as PubTypesId,
+					stageId: stageId as StagesId,
+					values: enabledPubValues as Record<string, any>,
```

Comment:
I don't like the cast to any here, but I wasn't sure how to get these types to match up 😕 

```
Type 'PubValues' is not assignable to type 'Record<string, Json>'.
  'string' index signatures are incompatible.
    Type 'JsonValue' is not assignable to type 'Json'.
      Type 'JsonObject' is not assignable to type 'Json'.
        Type 'JsonObject' is missing the following properties from type 'Json[]': length, pop, push, concat, and 35 more.ts(2322)
integrations.ts(97, 2): The expected type comes from property 'values' which is declared here on type 'CreatePubRequestBodyWithNullsNew | { id?: string | undefined; pubTypeId: string; parentId?: string | null | undefined; values: Record<string, Json>; assigneeId?: string | undefined; children?: (Omit<CreatePubRequestBodyWithNulls, "stageId"> & { stageId?: StagesId | undefined; })[] | undefined; stageId?: StagesId | undefined; }'
```

Reply:
I think this is fine for now! I've had trouble with the `Json` and `JsonValue` types before, too. At some point we can do a pass to improve the types for pub values objects across the project.